packages/code/src/Code/Code.tsx

```diff
@@ -163,6 +188,13 @@ function Code({
       : { usePortal }),
   } as const;
 
+  const collapsedLinesOfCode = 5;
```

Comment:
not: since this will always be 5, maybe we should make this a global constant since you're also using it in styles?

Reply:
This makes sense to m. Could you point me to where we store global constants. I see `lib/` seems to be similar but I'm not quite sure that's correct.

Reply:
You'll notice that some components have a `constants.ts` file — you can add a constants.ts file inside `code/src`.